assets/css/style.css

```diff
@@ -1,3 +1,276 @@
-.makeHeaderRed {
-  color: red;
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: calc(20px + 0.390625vw);
+}
+
+body {
+  background-color: black;
+}
```

Comment:
- [ ] The `background-color` used for the Figma template is `#1C1A19`. It is a lighter shade than the normal black `color`. Kindly use it so that your design is identical to that of the Figma template you chose. 👍